plugins/krakenExchange.go

```diff
@@ -9,7 +9,9 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Beldur/kraken-go-api-client"
+	"github.com/interstellar/kelp/support/logger"
+
+	krakenapi "github.com/Beldur/kraken-go-api-client"
```

Comment:
this import should not have changed: `krakenapi "github.com/Beldur/kraken-go-api-client"`

Reply:
how does the `apis                     []*krakenapi.KrakenApi` work in the KrakenExchange struct work if there's no `krakenapi` import?

Reply:
we import it without the alias, which seems to work. if you can't get that to work then the alias is ok